crates/binjs_io/src/entropy/read/content_decoders.rs

```diff
@@ -79,8 +85,20 @@ where
                     Ok(result) => result as usize,
                     Err(err) => return Some(Err(TokenReaderError::ReadError(err))),
                 };
-
                 debug!(target: "read", "DictionaryStreamDecoder::next index: {}", index);
+                let index = if index == 0 {
+                    // Special case: 0 is reserved for "next in prelude".
```

Comment:
to me, "next in prelude" isn't clear what it means (especially what the unit for "next" is)
can it be "next entry in prelude dictionary" or something?